### ISSUE TITLE
Keypad Chiming Zones

### ISSUE THREAD

**Issue by RaeesD** [ParadoxAlarmInterface/pai#370]
## Feature description
It would be great if the Chiming Zones of the keypads can be exposed as switches.

In BabyWare, via the keypad module's properties, it is currently possible to toggle zones to chime from the keypad when they open. If these toggles could be exposed as switches in PAI, we could switch these on or off, or automate according to a schedule in HA.

**Comment by yozik04** [comment]
Seems to be very narrow use case.

**Comment by karamfil** [comment]
Oh man, I would love this!

**Comment by beerygaz** [comment]
Like this idea too.  Mostly we turn the chime off, but when someone is home alone or in a vulnerable situation, turning chimes on via HA on a schedule would be awesome.

**Comment by clau-bucur** [comment]
Agreed also, this would be a nice addition for better scheduling and automations.

**Comment by yellowonblack** [comment]
+1

**Comment by yozik04** [comment]
Closing as **stale**: this issue has had no activity for more than 18 months and could not be reproduced or progressed with the information available.

This is a housekeeping pass, not a judgement on the merit of the request. If this is still relevant on a current PAI release, please comment and it will be reopened, or start a thread in [Discussions](https://github.com/ParadoxAlarmInterface/pai/discussions).
